tests/unit/amo/components/TestUserProfileEditPicture.js

```diff
@@ -101,22 +102,29 @@ describe(__filename, () => {
     sinon.assert.callCount(onSelect, 1);
   });
 
-  it('renders a "delete" button when user has a picture URL', () => {
+  it('renders a "delete" confirm button when user has a picture URL', () => {
```

Comment:
should we add a test for showing the default button ('Delete this picture') button too?

also should we get more specific and show a test that this (the delete confirm) only comes up after clicking the "Delete this picture" button? or is that just assumed?

Reply:
I guess I do see the delete-button below but maybe if the description is a little different that will be a little clearer ~ something like "renders a delete link which triggers a confirm (delete) button when user has a picture URL"

Reply:
I will change to `renders a "delete" ConfirmButton when user [...]` so that it indicates that we're using a component. Would that work for you?